packages/core/lib/debug/interpreter.js

```diff
@@ -628,7 +634,13 @@ class DebugInterpreter {
       case "l":
         if (this.session.view(session.status.loaded)) {
           this.printer.printFile();
-          this.printer.printState(LINES_BEFORE_LONG, LINES_AFTER_LONG);
+          // update the numbers of lines to be printed
+          const {beforeLines, afterLines} = this.updatePrintoutLines(
+            splitArgs,
+            this.printer.stateLines
+          );
+          this.printer.stateLines = [beforeLines, afterLines];
```

Comment:
Hm, this is different from the functionality we discussed on Slack, and this is the other reason I'm hitting "request changes".  This line of code means that commands like `l +5` will have a persistent effect.  Did you decide to do it differently?  Can you explain what your intention here was?

This sort of thing is why I really recommend using your initial PR writeup to describe in more detail the decisions you made when writing the PR.  You can use it to highlight decisions and tradeoffs, making it clear what downsides are deliberate tradeoffs, vs which are simply accidents or bugs.  It can be very helpful both to reviewers, and to later people who need to investigate the changes!

Reply:
No, I do not mean to do it differently.  Actually, parsePrintoutLines() need to do `if -/+ follows by space, continue`.

Reply:
OK I guess you've explained in the other comment why you changed it, so, I don't think we need to worry about this.